Fixing GS guide and links (#2186)
dbt-labs/docs.getdbt.com: website/docs/docs/build/sql-models.md

:::info Building your first models

> If you're new to dbt, we recommend that you check out our [Getting Started guide](/docs/get-started/overview) to build your first dbt project with models.

Author: ```suggestion
If you're new to dbt, we recommend that you read the [Getting Started guide](/docs/get-started/overview) to build your first dbt project with models.
```
